bitnami/external-dns/templates/pdb.yaml

```diff
@@ -20,8 +20,8 @@ spec:
   {{- if .Values.pdb.minAvailable }}
   minAvailable: {{ .Values.pdb.minAvailable }}
   {{- end }}
-  {{- if .Values.pdb.maxUnavailable }}
-  maxUnavailable: {{ .Values.pdb.maxUnavailable }}
+  {{- if or .Values.pdb.maxUnavailable (not .Values.pdb.minAvailable) }}
+  maxUnavailable: {{ .Values.pdb.maxUnavailable | default 1 }}
   {{- end }}
```

Comment:
Nice! I love this workaround, it'll definitely help with the UX. That said, please ensure we state in the `pdb.maxUnavailable` parameter description this new behavior